<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>410</width>
                <height>350</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>410</width>
                <height>350</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>410</width>
                <height>350</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QGroupBox" name="groupBox">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>10</y>
                        <width>391</width>
                        <height>91</height>
                    </rect>
                </property>
                <property name="title">
                    <string>Control</string>
                </property>
                <widget class="QComboBox" name="webcamSelection">
                    <property name="geometry">
                        <rect>
                            <x>10</x>
                            <y>30</y>
                            <width>111</width>
                            <height>31</height>
                        </rect>
                    </property>
                    <item>
                        <property name="text">
                            <string>Webcam 1</string>
                        </property>
                    </item>
                    <item>
                        <property name="text">
                            <string>Webcam 2</string>
                        </property>
                    </item>
                </widget>
                <widget class="QComboBox" name="gameSelection">
                    <property name="geometry">
                        <rect>
                            <x>140</x>
                            <y>30</y>
                            <width>111</width>
                            <height>31</height>
                        </rect>
                    </property>
                </widget>
                <widget class="QPushButton" name="startBtn">
                    <property name="geometry">
                        <rect>
                            <x>270</x>
                            <y>30</y>
                            <width>111</width>
                            <height>51</height>
                        </rect>
                    </property>
                    <property name="styleSheet">
                        <string notr="true"/>
                    </property>
                    <property name="text">
                        <string>Start</string>
                    </property>
                </widget>
            </widget>
            <widget class="QGroupBox" name="groupBox_2">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>120</y>
                        <width>171</width>
                        <height>161</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <weight>50</weight>
                        <bold>false</bold>
                    </font>
                </property>
                <property name="title">
                    <string>Status</string>
                </property>
                <property name="flat">
                    <bool>false</bool>
                </property>
                <property name="checkable">
                    <bool>false</bool>
                </property>
                <widget class="QLabel" name="label">
                    <property name="geometry">
                        <rect>
                            <x>10</x>
                            <y>50</y>
                            <width>121</width>
                            <height>16</height>
                        </rect>
                    </property>
                    <property name="font">
                        <font>
                            <pointsize>11</pointsize>
                        </font>
                    </property>
                    <property name="text">
                        <string>Game:</string>
                    </property>
                </widget>
                <widget class="QLabel" name="label_2">
                    <property name="geometry">
                        <rect>
                            <x>10</x>
                            <y>90</y>
                            <width>81</width>
                            <height>16</height>
                        </rect>
                    </property>
                    <property name="font">
                        <font>
                            <pointsize>11</pointsize>
                        </font>
                    </property>
                    <property name="text">
                        <string>FPS:</string>
                    </property>
                </widget>
                <widget class="QLabel" name="label_3">
                    <property name="geometry">
                        <rect>
                            <x>10</x>
                            <y>130</y>
                            <width>51</width>
                            <height>16</height>
                        </rect>
                    </property>
                    <property name="font">
                        <font>
                            <pointsize>11</pointsize>
                            <weight>50</weight>
                            <bold>false</bold>
                        </font>
                    </property>
                    <property name="text">
                        <string>Status:</string>
                    </property>
                </widget>
                <widget class="QLabel" name="statusLabel">
                    <property name="geometry">
                        <rect>
                            <x>70</x>
                            <y>130</y>
                            <width>101</width>
                            <height>21</height>
                        </rect>
                    </property>
                    <property name="font">
                        <font>
                            <pointsize>13</pointsize>
                            <weight>75</weight>
                            <bold>true</bold>
                        </font>
                    </property>
                    <property name="styleSheet">
                        <string notr="true">color: rgb(255, 85, 127);</string>
                    </property>
                    <property name="text">
                        <string>Stopped</string>
                    </property>
                </widget>
                <widget class="QLabel" name="selectedGameLabel">
                    <property name="geometry">
                        <rect>
                            <x>60</x>
                            <y>50</y>
                            <width>131</width>
                            <height>16</height>
                        </rect>
                    </property>
                    <property name="font">
                        <font>
                            <pointsize>10</pointsize>
                            <weight>75</weight>
                            <bold>true</bold>
                        </font>
                    </property>
                    <property name="styleSheet">
                        <string notr="true">color: rgb(77, 208, 225);</string>
                    </property>
                    <property name="text">
                        <string/>
                    </property>
                </widget>
                <widget class="QLabel" name="fpsLabel">
                    <property name="geometry">
                        <rect>
                            <x>50</x>
                            <y>90</y>
                            <width>81</width>
                            <height>16</height>
                        </rect>
                    </property>
                    <property name="font">
                        <font>
                            <pointsize>15</pointsize>
                            <weight>75</weight>
                            <bold>true</bold>
                        </font>
                    </property>
                    <property name="styleSheet">
                        <string notr="true">color: rgb(77, 208, 225);</string>
                    </property>
                    <property name="text">
                        <string>0</string>
                    </property>
                </widget>
            </widget>
            <widget class="QLabel" name="imageLabel">
                <property name="geometry">
                    <rect>
                        <x>200</x>
                        <y>120</y>
                        <width>201</width>
                        <height>171</height>
                    </rect>
                </property>
                <property name="styleSheet">
                    <string notr="true">*{
                        border: 4px solid rgb(77, 208, 225);
                        border-radius: 7px;
                        }
                    </string>
                </property>
                <property name="text">
                    <string/>
                </property>
                <property name="pixmap">
                    <pixmap>../Pictures/Camera Roll/WIN_20181012_17_51_53_Pro.jpg</pixmap>
                </property>
                <property name="scaledContents">
                    <bool>true</bool>
                </property>
                <property name="alignment">
                    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
                </property>
                <property name="wordWrap">
                    <bool>false</bool>
                </property>
            </widget>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>410</width>
                    <height>21</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
